homeassistant/components/lovelace/websocket.py

```diff
@@ -86,9 +87,9 @@ async def websocket_lovelace_config(
     connection: websocket_api.ActiveConnection,
     msg: dict[str, Any],
     config: LovelaceStorage,
-) -> None:
```

Comment:
This was very confusing because its typed to `None` but it returns the config payload